src/tunnel.js

```diff
@@ -58,6 +88,16 @@ exports.transformUrl = function(origUrl, host, tunnelHost) {
   return newUrl;
 };
 
-exports.disconnect = function() {
-  ngrok.disconnect();
+exports.disconnect = function(response) {
+  return new Promise(function(resolve) {
+    if (sauceConnection) {
+      sauceConnection.close(function() {
+        console.log('Closed Sauce Connect process');
```

Comment:
Let's remove this log, and add back the general tunnel disconnecting log above

Reply:
removed